juriscraper/lib/date_utils.py

```diff
@@ -154,3 +154,17 @@ def is_first_month_in_quarter(month):
     :return: Whether that month is the first month in a quarter
     """
     return month in [1, 4, 7, 10]
+
+def fix_future_year_typo(future_date):
+    """Fix year typo, convert 2106 to 2016"""
+    year = str(future_date.year)
+    if year[2] == '0':
```

Comment:
I'm not quite happy with this implementation yet. It feels like it could cause unwanted changes. We really want to test just for the two middle digits being transposed, so maybe we should check for explicitly that. For example:

    current_year = str(date.today().year)
    transposed_year = current_year[0] + current_year[2] + current_year[1] + current_year[3]
    if transposed_year == str(future_date.year):
        # fix it.

Something like that will only fix problems in the current scrapers, but I like that it is pretty limited in the damage it could cause. What do you think?